dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/userdata/AuthenticatedUserCsvWriter.java

```diff
@@ -40,7 +40,9 @@ public void write(OutputStream outputStream, List<AuthenticatedUser> authenticat
 
             csvPrinter.printRecord(AuthenticatedUserCSVRecord.getHeaders());
             for(AuthenticatedUser user : authenticatedUsers) {
-                csvPrinter.printRecord(buildRecord(user).getValues());
+                if(! user.isErased()) {
```

Comment:
Same as above, this seems to be the wrong place to filter out records.

Reply:
fixed